shoulder/config.py

```diff
@@ -134,6 +134,10 @@ def validate(self):
 # Function generation options
 # -----------------------------------------------------------------------------
 
+c = Configuration("encoded_functions", True)
```

Comment:
What do you think about defaulting this to False until the C++ header is updated to support these new accessors?

Reply:
Yeah, I agree with that. I forgot about the C++ generator. I set this to False by default and I will work on integrating these changes with the C++ generator.